vignettes/goshawk.Rmd

```diff
@@ -0,0 +1,84 @@
+---
+title: "Introduction to goshawk"
+date: "2022-03-09"
+output:
+    rmarkdown::html_document:
+        theme: "spacelab"
+        highlight: "kate"
+        toc: true
+        toc_float: true
+vignette: >
+    %\VignetteIndexEntry{Introduction to goshawk}
+    %\VignetteEngine{knitr::rmarkdown}
+    %\VignetteEncoding{UTF-8}
+editor_options:
+    markdown: 
+        wrap: 72
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+---------
+
+## Introduction to goshawk
+
+---------
+
+This vignette shows the general purpose and syntax of the `goshawk` R package.  
+The `goshawk` R package contains analytical functions for primarily creating longitudinal visualizations useful for clinical trials and other statistical analysis. 
+
+---------
+
+## Common Clinical Trials Analyses
+
+The package provides several functions to create graphs used for clinical trial and other statistical analysis.
```

Comment:
I think a couple of words here should be plural.

'trial' -> 'trials'

'analysis' -> 'analyses'